cypress/integration/configuration/attributes/createContentAttribute.js

```diff
@@ -109,7 +125,7 @@ filterTests({ definedTags: ["all"] }, () => {
       });
     });
 
-    it("should create attribute without required value", () => {
+    it("should be able to create attribute without require value TC:SALEOR_0522", () => {
```

Comment:
`requiring?`

Reply:
I used title assigned to this test our documents  "Admin should be able to create attribute without require value"